docs/howto/remote-debugging.rst

```diff
@@ -0,0 +1,106 @@
+======================================
+How to debug an app in iOS and Android
```

Comment:
I feel this title should specify that this tutorial is Pycharm and iOS only: `How to debug an iOS app from PyCharm`

Note: I'm not a toga maintainer nor anything, just a random internet person looking through the issues :)